src/main/java/org/junit/rules/ExternalResource.java

```diff
@@ -44,11 +48,20 @@ private Statement statement(final Statement base) {
             @Override
             public void evaluate() throws Throwable {
                 before();
+
+                List<Throwable> errors = new ArrayList<Throwable>();
                 try {
                     base.evaluate();
+                } catch(Throwable t) {
```

Comment:
Style-nit : please put a space after "catch" (we use Google Java style which requires spaces around keywords like "if" and "catch"